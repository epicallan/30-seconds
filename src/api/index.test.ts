import * as prettyFormat from 'pretty-format';
// import {gqlFileContent} from '.';

describe('API tests', () => {
    it('should work', async () => {
        const data = {a: 'hey'};
        expect(prettyFormat(data)).toMatchSnapshot();
     });
});
