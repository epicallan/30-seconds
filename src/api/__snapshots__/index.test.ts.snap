// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`API tests should work 1`] = `
"Object {
  \\"a\\": \\"hey\\",
}"
`;
